librz/core/core.c

```diff
@@ -2447,30 +2447,62 @@ RZ_API RzCmdStatus rz_core_core_plugins_print(RzCore *core, RzCmdStateOutput *st
 	return RZ_CMD_STATUS_OK;
 }
 
+/**
+ * \brief Executes a command and returns the filtered output.
+ *
+ * \param core Pointer to RzCore instance.
+ * \param command Command to be executed.
+ * \param filter String used to filter the output.
+ * \return Filtered output of the command. Must be freed by the caller.
+ */
+RZ_API char *rz_core_filter_command_output(RZ_NONNULL RzCore *core, const char *command, const char *filter) {
+	rz_return_val_if_fail(core, NULL);
+	char *output = rz_core_cmd_str(core, command);
+	char *filtered_output = NULL;
+
+	RzList *lines = rz_str_split_list(output, "\n", 0);
+	RzListIter *iter;
+	char *line;
+	rz_list_foreach (lines, iter, line) {
+		if (strstr(line, filter)) {
+			if (filtered_output) {
+				filtered_output = rz_str_append(filtered_output, "\n");
+			} else {
+				filtered_output = rz_str_dup("");
+			}
+			filtered_output = rz_str_append(filtered_output, line);
+		}
+	}
+
+	RZ_FREE(output);
+	return filtered_output;
+}
+
 /**
  * \brief Seeks to any basic block of the current function.
  *
  * \param core The RzCore instance.
  */
-RZ_API void rz_core_seek_bb(RZ_NONNULL RzCore *core) {
+RZ_API void rz_core_view_and_seek_to_bb(RZ_NONNULL RzCore *core) {
```

Comment:
also this should not be an api but should be in `tui/visual.c`

Reply:
yup, was going to do that, wanted a code review before shifting it to `visual.c`.